base/applications/regedit/treeview.c

```diff
@@ -798,30 +798,26 @@ BOOL SelectNode(HWND hwndTV, LPCWSTR keyPath)
         size_t copyLength;
         s = wcschr(keyPath, L'\\');
         if (s != NULL)
-        {
```

Comment:
You trying to start an edit war over this?

Reply:
I can change that back if you want.

Reply:
I added them back. Resolved.

Reply:
Thank you.